spaceship/lib/spaceship/connect_api/models/user.rb

```diff
@@ -62,6 +62,11 @@ def get_visible_apps(client: nil, limit: nil)
         resp = client.get_user_visible_apps(user_id: id, limit: limit)
         return resp.to_models
       end
+
+      def delete!(client: nil)
```

Comment:
Could this method be moved above `get_visible_apps`?

Reply:
Done 🙂